### DISCUSSION TITLE
Where should cases be created for NorESM2.5 developement?

### DISCUSSION THREAD

**Discussion by mvertens** [NorESMhub/noresm3_dev_simulations#11] [Q&A] [answered]
We agreed that cases should be placed in a common project area. Also - will cases appear in each user's directory under this common project space?
@adagj @matsbn  - did we agree on an exact place and who will be responsible for creating it and ensuring that permissions are appropriate for us all to share this space.

**Comment by adagj** [comment]
We didn't agree on that. During the core meeting, we agreed that the cases should not be stored in a common directory. However, the case folders need to be archived together with the hist files and all output in the common directory on NIRD: /projects/NS9560K/noresm3/cases/

**Reply by mvertens** [reply]
@adagj - thanks for the clarification. I mixed up case creation and running with archiving. So according to this - cases can be created anywhere the user wants but archived in a specific location. Do I have that right?

**Reply by adagj** [reply, marked as answer]
Cases need to be created in **project areas** where we all have access so it is possible to look at each others cases. Users should not create NorESM2.5 development cases in private directories.  We recommend nn9560k. But we don't need to have one folder where we all create cases.

**Reply by mvertens** [reply]
okay - that makes sense. Thanks.
